index.html

```diff
@@ -13,21 +13,21 @@
     <nav>
       <div class="nav-bar">
         <a href="#" class="logo">Kajal</a>
-        <button class="hambug-menu">
+        <button type="button" class="hambug-menu">
          <a href="#"><img src="./img/menu.svg" alt="menu"></a>
         </button>
       </div>
     </nav>
 
     <section class="head-section">
       
-      <h1 class="header-title">I'm Kajal Pramanik to see you!</h1>
+      <h1 class="title">I'm Kajal Pramanik to see you!</h1>
       
       <p class="header-text">
         I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
       </p>
       
-      <p class="header-social-text">Let's connect</p>
+      <p class="page-social-text">Let's connect</p>
```

Comment:
- [ ]Great job so far, however, there are a few minor details you should work on. As needed for this project, you should use `hx` for all sub-titles rather than a `p` tag; please modify this for your code to appear professional and have proper semantics. :wink: